pkg/transaction/snapper.go

```diff
@@ -203,7 +203,10 @@ func (sn *snapperT) probe(d deployment.Deployment) (err error) {
 
 	part := sn.hwPartitions.GetByUUIDNameOrLabel(sysPart.UUID, sysPart.Role.String(), sysPart.Label)
 	if part == nil {
-		return fmt.Errorf("system partition not found: %+v", sysPart)
+		part, err = block.GetPartitionByUUID(sn.s, blockDev, sysPart.UUID, 4)
```

Comment:
The first version made by @frelon completely replaced `GetByUUIDNameOrLabel` call by `GetPartitionByUUID`, it works but I had failures in the Unit tests and I didn't find how to fix them...

This method makes the tests happy and also I think it's better to use use `GetPartitionByUUID` only when needed.

Reply:
As we discussed: removed!